### ISSUE TITLE
Remove CSS styles that assign text color

### ISSUE THREAD

**Issue by knice** [ucsc/ucsc-gutenberg-blocks#131]
CSS styles in the plugin's CSS file override the theme styles. Several of the color combinations (text color and background) have been flagged by Siteimprove as not meeting WCAG AA requirements for contrast. 

Removing assigned text colors from the plugin's CSS will let the theme's styles take precedence and resolve the a11y contrast issues.

**Comment by HenryHeikkinenUCSC** [comment]
https://ucsc-its.atlassian.net/browse/CPM-17

**Comment by HenryHeikkinenUCSC** [comment]
Would this be for all three custom blocks - class schedule, course catalog and campus directory, or is this ticket for just one of the three? It can be split into 3 in Jira dont need to do that here.

**Comment by ghagenah** [comment]
Hi, @knice. In the context of the Course Catalog, when we remove the styling that sets the color to UCSC Primary Blue, the table headers default to the color set in the body: black. The UCSC Blue (#003C6C) has a color contrast ratio of 11.27:1 which is sufficient for WCAG AA & AAA. Let me know if we should go ahead and make this change, or if you can point us to specific contrast issues with Course Catalog or the other modules.

Here's the change we can make in tablesorter.css for reference:

```
.table-sortable thead th {
    text-align: left;
    border-bottom: 1px solid black;
    /* color: var(--wp--preset--color--ucsc-primary-blue); */
}
```
